File: c-plus-plus/URI_1035.cpp
#include <iostream>
using namespace std;
int main () {
	int a, b, c, d;
	cin >> a >> b >> c >> d;
	bool aceitos = false;
	if (b > c)
	{
		if (d > a)
		{
			if ((c + d) > (a + b))
			{
				if (c >= 0 && d >= 0)
				{
					if (a % 2 == 0)
					{
						aceitos = true;
					}
				}
			}
		}
	}
	if (aceitos)
	{
		cout << "Valores aceitos" << endl;
	}
	else
	{
		cout << "Valores nao aceitos" << endl;
	}
	
	
	return 0;
}

File: ruby/URI_1035.rb
valores = gets.split

a = valores[0].to_i
b = valores[1].to_i
c = valores[2].to_i
d = valores[3].to_i

if b > c && d > a && (c + d) > (a + b) && c >= 0 && d >= 0 && a % 2 == 0 then
	puts "Valores aceitos"
else
	puts "Valores nao aceitos"
end